lib/src/default_emoji_picker_view.dart

```diff
@@ -155,44 +161,47 @@ class _DefaultEmojiPickerViewState extends State<DefaultEmojiPickerView>
       return _buildNoRecent();
     }
     // Build page normally
-    return GridView.count(
-      scrollDirection: Axis.vertical,
-      physics: const ScrollPhysics(),
-      controller: _scrollController,
-      shrinkWrap: true,
-      primary: false,
-      padding: const EdgeInsets.all(0),
-      crossAxisCount: widget.config.columns,
-      mainAxisSpacing: widget.config.verticalSpacing,
-      crossAxisSpacing: widget.config.horizontalSpacing,
-      children: categoryEmoji.emoji.asMap().entries.map((item) {
-        final index = item.key;
-        final emoji = item.value;
-        final onPressed = () {
-          _closeSkinToneDialog();
-          widget.state.onEmojiSelected(categoryEmoji.category, emoji);
-        };
+    return GestureDetector(
+      onTap: _closeSkinToneDialog,
```

Comment:
Can you explain in which situation this is needed? I've tested it but it was never triggered when the skinColorDialog was visible, because once you scroll or swipe it will close anyway 🤔

Reply:
Thanks for your response! My first attempt at contributing to open source projects actually :)! 

Yeah you can view this as an additional option to closing the overlay, that is in addition to swiping and scrolling. If you had not mentioned it, I didn't know that scrolling or swiping closes the overlay. My default expectation was that tapping on an empty space within the emoji keyboard should close any existing overlay - and I think other users may have that thought too. So this doesn't prevent closing by swiping and scrolling, it just adds to number of ways in which the overlay can be dismissed. I first noticed it when in the "recently used emoji category" there weren't many options and I had a skin tone overlay displaying, I tried tapping in the empty space to close it but it wasn't working, hence I added this particular piece. Lemme know if it's still not clear, so I could send maybe a video demo of what I meant. 
Cheers.

Reply:
I see, thanks for the explanations, makes absolute sense ! I've tested and confirmed it. Thanks again for your contribution and I hope you will make more contributions to open source from now on :)